src/http.js

```diff
@@ -29,7 +29,7 @@ const agent = new https.Agent({
 const tokenConfig = {
     httpsAgent : agent,
     method: 'post',
-    url: config.keycloakServerBaseURL + '/auth/realms/' + config.adminRealm + '/protocol/openid-connect/token',
```

Comment:
ok rimuovere il prefisso `/auth` da tutti i path, però inseriamo indicazione in `.env-example` (non sono riuscito a inserirlo io nella PR stessa, non so come si faccia)

da 

    keycloakServerBaseURL = http://localhost:8080

a 

    keycloakServerBaseURL = http://localhost:8080/auth

Reply:
ok inserisco indicazione in doc; per quanto riguarda "forceAuthn", che era la più controversa ho ricontrollato la doc (https://docs.italia.it/italia/spid/spid-regole-tecniche/it/stabile/single-sign-on.html) e conferma se ho capito bene: se il livello è 1 allora false, se 2 o 3 allora true, corretto?

Reply:
confermo - per quello lascerei come era prima perchè così prendiamo tutti i casi